web_src/js/features/projects.js

```diff
@@ -5,52 +5,115 @@ export default async function initProject() {
     return;
   }
 
-  const {Sortable} = await import(/* webpackChunkName: "sortable" */'sortablejs');
-  const boardColumns = document.getElementsByClassName('board-column');
+  const {Sortable} = await import(
+    /* webpackChunkName: 'sortable' */ 'sortablejs'
+  );
 
-  for (const column of boardColumns) {
-    new Sortable(
-      column.getElementsByClassName('board')[0],
-      {
-        group: 'shared',
-        animation: 150,
-        onAdd: (e) => {
-          $.ajax(`${e.to.dataset.url}/${e.item.dataset.issue}`, {
-            headers: {
-              'X-Csrf-Token': csrf,
-              'X-Remote': true,
-            },
-            contentType: 'application/json',
-            type: 'POST',
-            error: () => {
-              e.from.insertBefore(e.item, e.from.children[e.oldIndex]);
-            },
-          });
-        },
+  const colContainer = document.getElementById('board-container');
+  let projectURL = '';
+  let projectID = 0;
+  if (colContainer && colContainer.dataset) {
+    projectURL = colContainer.dataset.url;
+    projectID = colContainer.dataset.projectid;
+  }
+
+  // search as you type new issues
+  $('#current-card-details-input').on('keyup', '#issue-search', (e) => {
+    const q = e.currentTarget.value;
+    const repoURL = $('[data-repourl]').data('repourl');
+    fetch(`/api/v1/repos${repoURL}/issues?state=open&q=${q}&exclude_project_id=${projectID}`, {
+      method: 'GET',
+      headers: {'X-Csrf-Token': csrf, 'Content-Type': 'application/json'}
+    })
+      .then((res) => {
+        return res.json();
+      })
+      .then((data) => {
+        let cards = '';
+        data.map((issue) => {
+          const card = `<div class='card draggable-card board-card' data-priority='0' data-id='0' data-issueid='${issue.id}'>
+          <div class='content'>
+            <div class='header'>
+                    <a href='${repoURL}/${issue.IsPull ? 'pulls' : 'issues'}/${issue.number}/sidebar/true'
+                      data-url='${repoURL}/${issue.IsPull ? 'pulls' : 'issues'}/${issue.number}/sidebar/true'>
+            #${issue.number} ${issue.title}
+            </a>
+            </div>
+            <div class='meta'>
+            </div>
+            <div class='description'>
+
+            </div>
+          </div>
+          </div>`;
```

Comment:
Would suggest something like [detent.js](https://github.com/dmnd/dedent) to tame these inline HTML blocks:

````js
const card = detent`
  <div class='card draggable-card board-card' data-priority='0' data-id='0' data-issueid='${issue.id}'>
    <div class='content'>
      <div class='header'>
        <a href='${repoURL}/${issue.IsPull ? 'pulls' : 'issues'}/${issue.number}/sidebar/true' data-url='${repoURL}/${issue.IsPull ? 'pulls' : 'issues'}/${issue.number}/sidebar/true'>#${issue.number} ${issue.title}</a>
      </div>
      <div class='meta'></div>
      <div class='description'></div>
    </div>
  </div>
`;
````

Or maybe go one step further and [use JSX](https://github.com/vadimdemedes/dom-chef).

Reply:
Alternatively do it just like above without detent. I guess I can refactors these later to something better than plain template strings.

Reply:
@silverwind seems a bit out of scope for this PR, maybe in another PR?

Reply:
Yes indeed, but proper 2-space indendation should be in scope of this PR 😉.

Reply:
good point, pushed a fix thanks!